test/util/nightwatch-commands/focusOn.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Change select value and trigger change event programatically. This
+ * is necessary because long select boxes tend to render offscreen,
+ * causing Selenium to fail in unexpected ways.
+ * The first parameter is the field name, not the whole selector.
```

Comment:
Should this specify that the first parameter is the whole selector?

Reply:
Heh, woops! That comment is from another command that I just copied and gutted. I need to rewrite the whole thing. Good catch!